ios/GoodDollar.xcodeproj/project.pbxproj

```diff
@@ -728,7 +736,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = org.gooddollar;
 				PRODUCT_NAME = GoodDollar;
-				PROVISIONING_PROFILE_SPECIFIER = "GoodDollar Ad Hoc";
+				PROVISIONING_PROFILE_SPECIFIER = "GoodDollar App Store";
```

Comment:
should this be Ad Hoc regularly? and only switched on release to App Store?

Reply:
yes, I'll change back to ad hoc. it isn't a biggy but is preferable to keep ad hoc as the default